src/bot/index.ts

```diff
@@ -133,8 +133,8 @@ bot.on('Room.timeline', (event: mSDK.MatrixEvent) => {
     }
 
     // Parity users can override the drip amount by using a 3rd argument
-    if (sender.endsWith(':matrix.parity.io') && arg1) {
-      dripAmount = Number(arg1);
+    if (arg1 && isAccountPrivileged(sender)) {
+      dripAmount = Number(arg1) || defaultDripAmount;
```

Comment:
converting user-input string could endup with NaN, so defaulting it to standard amount

Reply:
I think it's better to send an error message in this case. "Defaulting to standard amount" is a server-side implementation detail which users aren't aware of.

Reply:
You're right, done